Model/Source/DigitalStampWeightOptions.php

```diff
@@ -46,15 +47,11 @@ public function __construct(Data $helper)
      *
      * @return bool
      */
-    public function getDefault($option)
+    public function getDefault($option): bool
     {
-        $settings = self::$helper->getStandardConfig('options');
+        $settings = self::$helper->getStandardConfig(CarrierPostNL::NAME, 'options');
```

Comment:
Hier ook bpost als default? Of is het postnl vanwege digital stamp? (Bestaat dat in België?)

Reply:
Nee bestaat niet daarom is het ook niet meegenomen in de .xml